openhabian-setup.sh

```diff
@@ -10,19 +10,6 @@
 #
 
 
-# Find the absolute script location dir (e.g. BASEDIR=/opt/openhabian)
-SOURCE="${BASH_SOURCE[0]}"
-while [ -h "$SOURCE" ]; do
-  BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-  SOURCE="$(readlink "$SOURCE")"
-  [[ $SOURCE != /* ]] && SOURCE="$BASEDIR/$SOURCE"
-done
-BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-SCRIPTNAME="$(basename "$SOURCE")"
-
-REPOSITORYURL="https://github.com/openhab/openhabian"
-CONFIGFILE="/etc/openhabian.conf"
-
```

Comment:
**if you want to factor this out, remember sourcing init.sh here!**
otherwise this will break openhabian-config!

Reply:
you mean sourcing the newly introduced init.bash ?
It is sourced from build-image/first-boot.bash